src/encoded/static/components/objectutils.js

```diff
@@ -910,6 +910,33 @@ export const isFileVisualizable = (file) => {
         && ['released', 'in progress', 'archived'].indexOf(file.status) > -1;
 };
 
+function isVisualizableAssembly(assembly) {
+    const visualizableAssemblies = [
+        'GRCh38',
+        'hg19',
+        'GRCh37',
+        'GRCm39',
+        'mm10',
+        'mm10-minimal',
+        'GRCm38',
+        'mm9',
+        'GRCm37',
+        'dm6',
+        'dm3',
+        'ce11',
+        'ce10',
+    ];
+
+    return visualizableAssemblies.includes(assembly);
+}
+
+function filterForVisualizableAssembly(fileList) {
+    if (fileList?.length === 0) {
+        return [];
+    }
+
+    return fileList.filter((file) => isVisualizableAssembly(file.assembly));
```

Comment:
You could, at least conceptually, crash here since — if `fileList` isn’t defined — the first if fails, but now you call `filter()` on an undefined `fileList`. The version of this under `filterForVisualizableFiles` doesn’t have this issue.